packages/image_picker/README.md

```diff
@@ -17,6 +17,17 @@ Add the following keys to your _Info.plist_ file, located in `<project root>/ios
 * `NSCameraUsageDescription` - describe why your app needs access to the camera. This is called _Privacy - Camera Usage Description_ in the visual editor.
 * `NSMicrophoneUsageDescription` - describe why your app needs access to the microphone, if you intend to record videos. This is called _Privacy - Microphone Usage Description_ in the visual editor.
```

Comment:
Since there's a text version below now, these should use the XCode UI labels rather than the raw keys (as in the Camera README you referenced), and the wording should change accordingly (e.g., "Add the following rows" rather than "Add the following keys").